docs/BedWars1058/hooks/vault-hook.md

```diff
@@ -24,4 +24,14 @@ chat:
   global: '{vPrefix}&6[SHOUT] {team} &7{player}&f{vSuffix}: {message}'
   team: '{vPrefix}&f{level}{team}&7 {player}{vSuffix} {message}'
   spectator: '{vPrefix}&7[SPECTATOR] {player}{vSuffix}: {message}'
-```
\ No newline at end of file
+```
+
+## Economy
+**BedWars1058** is also providing Vault Economy plugins support since the v21.11.1.
```

Comment:
Economy plugins were supported long before 21.11.1

Reply:
i watched version history from spigotMC and i saw that with 21.11.1 eco support got added.